### ISSUE TITLE
WebDriver instance (Chrome) cannot be run via TeamCity

### ISSUE THREAD

**Issue by IgorProtsenko** [angular/protractor#3480]
Hi
I'm trying to run protractor tests using TeamCity command line.

**Without CI it works just fine**, if it's CI process stuck in here:

```
[13:21:50]Step 7/7: Run tests (Command Line) (1m:03s)
[13:21:50][Step 7/7] Starting: C:\TeamCity\buildAgent\temp\agentTmp\custom_script7054838070526284400.cmd
[13:21:50][Step 7/7] in directory: C:\TeamCity\buildAgent\work\687131def64767d6\test\e2e
[13:21:52][Step 7/7] 
[13:21:52][Step 7/7] > sirv_tests@0.0.1 test C:\TeamCity\buildAgent\work\687131def64767d6\test\e2e
[13:21:52][Step 7/7] > protractor configs/protractor_config.js --troubleshoot:true
[13:21:52][Step 7/7] 
[13:21:53][Step 7/7] [13:21:53] I/direct - Using ChromeDriver directly...
[13:21:53][Step 7/7] [13:21:53] I/launcher - Running 1 instances of WebDriver
[13:22:53][Step 7/7] [13:22:53] E/launcher - "process.on('uncaughtException'" error, see launcher
[13:22:53][Step 7/7] [13:22:53] E/launcher - Process exited with error code 199
[13:22:53][Step 7/7] npm ERR! Test failed.  See above for more details.
[13:22:53][Step 7/7] Process exited with code 1
[13:22:53][Step 7/7] Step Run tests (Command Line) failed
```

where 
`[13:22:53][Step 7/7] [13:22:53] E/launcher - "process.on('uncaughtException'" error, see launcher`

seems to be timeout exception, since browser window does not appear and nothing happens after
`[13:21:53][Step 7/7] [13:21:53] I/launcher - Running 1 instances of WebDriver`

Windows 7 64bit, last or 4.0.0 Protractor, last TeamCity build.

What I've tried:
Change Java to 7.*  Env variable points to 7.*;
Define Chromedriver location thought browser capabilities;
With ot without directConnect key;
Define WebDriver url in config and run "webdriver-manager update/start" as preparation step.

Whatever way I did that,  protractor config --troubleshoot doesn't work for me (options: **--troubleshoot, --troubleshoot true, --troubleshoot:true**  - no effect).

My package:
```
{
  "name": "tests",
  "version": "0.0.1",
  "description": "",
  "keywords": [
    "testing",
    "e2e",
    "protractor",
    "angular"
  ],
  "scripts": {
    "test": "protractor configs/protractor_config.js --troubleshoot true"
  },
  "dependencies": {
    "protractor": "4.0.0",
    "protractor-jasmine2-html-reporter": "0.0.6",,
    "path": "latest",
  },
  "author": "",
  "license": "MIT"
}
```

Current config (one of):
```
var path = require("path"), config;

config = {
 //   seleniumAddress: 'http://localhost:4444/wd/hub',
    specs: ["../tests/ui/Tests.js"],
    chromeOnly: true,
    directConnect: true,
    capabilities: {
        "browserName": "chrome"
/*        "chromeOptions": {
            binary: 'C:\\chromedriver.exe',
            args: [],
            extensions: []
        }*/
    },
    framework: "jasmine2",
    allScriptsTimeout: 100000,

    onPrepare: function() {
        browser.driver.manage().window().maximize();
        browser.manage().timeouts().implicitlyWait(1000);
        browser.driver.manage().timeouts().setScriptTimeout(100000);
    },

    jasmineNodeOpts: {
        showColors: true,
        onComplete: null,
        isVerbose: true,
        includeStackTrace: true,
        stopSpecOnExpectationFailure: false,
        defaultTimeoutInterval: 100000
    }
};

exports.config = config;

```

tests are run either with "node test" or "protractor _config_name". Both works if it's not CI.

Any concerns?

**Comment by NickTomlin** [comment]
Hi there!

Your question is better suited for StackOverflow or Gitter. Please [ask a question there with the 'protractor' tag](http://stackoverflow.com/questions/ask?tags=protractor) or post in the [Gitter Channel](https://gitter.im/angular/protractor) to get help.

From the the [getting help](https://github.com/angular/protractor#getting-help) section of the README:

> Please ask usage and debugging questions on StackOverflow (use the "protractor" tag) or in the Angular discussion group. (Please do not ask support questions here on Github.)

Thanks!

**Comment by cnishina** [comment]
I agree with the comment above to visit the Gitter channel. The exception has been fixed, please use `"protractor": "^4.0.3"`... if you update your Protractor version, you should be able to see the full exception.
